tests/unit/ssg-module/test_playbook_builder_data/guide/configure_crypto_policy/rule.yml

```diff
@@ -81,6 +81,13 @@ ocil: |-
     <pre>$ ls /etc/crypto-policies/local.d/ | awk -F'-' '{print $1}' | uniq | sort</pre>
     Outputs of two previous commands should match.
 
+fixtext: |-
+    Configure {{{ full_name }}} to implement approved encryption.
+    
+    update-crypto-policies --set "{{{ xccdf_value("var_system_crypto_policy") }}}"
```

Comment:
We have to remove the double quotes here since some variables already have them, which I believe it will make it have twice the double quotes: https://github.com/ComplianceAsCode/content/blob/14f10f513ad32d4f9c23618192a7007ad58d1f80/linux_os/guide/system/software/integrity/crypto/var_system_crypto_policy.var#L17